### PR TITLE
2/bots commands

### LINKED ISSUE #2: Telegram bot: Notify standing orders
See RF.6.2

Includes:
- [X] Create echo bot: https://github.com/gnosis/dex-telegram/pull/4
- [X] Add hot reload for development: https://github.com/gnosis/dex-telegram/pull/5
- [X] Docker: https://github.com/gnosis/dex-telegram/pull/6
- [X] Travis: https://github.com/gnosis/dex-telegram/pull/7
- [ ] Listen to Ethereum Events
- [ ] Notify standing orders

**Estimation**: 3

### PR DESCRIPTION
**Pull request by anxolin**
Closes #2

- [X] Improves how to handle messages. Now it replies to all messages with the help response text
- [X] Detects when the user executes a command: Eithers takes care of the command, or it shows that it doesn't recognize the comand
- [X] Adds a command `/start` and `/help` that would display the help response text
- [X] Adds a command `/about` with some information about the bot, including info about the connected network, node info, last minded block and the version of the bot iself
- [X] Subscribes to the events of new sell orders and notify in the channel

The bot output should look something like:
image

The bot will respond to the commands:
image

### CODE REVIEW

File: src/bot.ts [hunk 1]
```diff
@@ -13,39 +14,104 @@ const channelId = process.env.TELEGRAM_CHANNEL_ID as string
...
+
+Also, you can ask about me by using the command: /about`
+  )
+}
+
+async function _aboutCommand (msg: Message) {
+  const [blockNumber, networkId, nodeInfo] = await Promise.all([
+    dfusionRepo
+      .getBlockNumber()
+      .then(_toString)
```
Comment (Velenir) [inline src/bot.ts]: consider using `.then(String)` as it's [more reliable](https://devdocs.io/javascript/global_objects/string#String_conversion)

File: tsconfig.json [hunk 2]
```diff
@@ -10,6 +10,7 @@
     "moduleResolution": "node",
     "allowJs": false,
     "checkJs": false,
+    "resolveJsonModule": true,
```
Comment (Velenir) [inline tsconfig.json]: JSON types ftw :+1:

File: package.json [hunk 3]
```diff
@@ -4,7 +4,7 @@
   "description": "",
   "scripts": {
     "start": "DEBUG=ERROR-*,WARN-*,INFO-* node build/bot",
```
Comment (Velenir) [inline package.json]: You sure you don't need to require `dotenv/config` here too?

Comment (anxolin) [reply package.json]: I was considering it, but I think for production we'll use process env vars for sure, so not needed.

It doesn't harm to add it anyways, what do you think?

Comment (Velenir) [reply package.json]: not needed then

File: src/bot.ts (hunk 1 above)
Comment (W3stside) [reply src/bot.ts]: uhhh wait but 

```
String(null)
// "null"
```

is that what we want?

Comment (anxolin) [reply src/bot.ts]: I think 'null' there it's fine, just to show in the about message. See the picture in description

Comment (anxolin) [reply src/bot.ts]: I guess, normally we will get the block

**Pull request merged**